### DISCUSSION TITLE
Dashboard as a separate nuget package, rather than a docker image

### DISCUSSION THREAD

**Discussion by Nabakamal** [microsoft/aspire#4395] [Q&A] [answered]
Hello, there. I see that the dashboard is available as a standalone docker image (https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/dashboard/standalone?tabs=bash) but is it possible to run the dashboard as a separate project rather than being deployed as a container to connect to? 

Something along the lines of:
var dashboard = _dashboardService.InitializeDashboard(o => new AspireDashboardOptions()).GetDashboard();
dashbaord.StartAsync();

**Comment by davidfowl** [comment]
No there's no nuget package for the dashboard. Why do you want to run it as a nuget package in a project?

**Reply by Nabakamal** [reply]
Hi, David - thanks for the quick response. So, we've got (obviously non-Aspire) projects deployed on-prem right now and I just want to be able to use/test this dashboard without having to run it from within Docker(in the server). Alternatively, having had a quick think about it, an option to run it "explicitly" like here: https://www.metabase.com/start/oss/ also seems exciting. 

Here's another example of the type of implementation I'm referring to: https://github.com/lucaslorentz/durabletask-extensions/tree/main/src/LLL.DurableTask.Ui. Hope this answers your question. If you can advise an alternative way of getting this on an on-prem server, I'm all ears. Great work by you and the team, btw!  Cheers!

**Reply by davidfowl** [reply]
You can install the workload and run the executable manually as an alternative. Installing docker or podman in the on prem server would be a much better investment IMHO.

**Reply by Nabakamal** [reply]
I agree. However, being behind a corporate firewall means that we have a few restrictions to adhere to; getting there :) Do you have a link handy to do the "install the workload and run the executable manually" bit? If not, I can find out. Thanks for being so responsive & helpful - cheers!

**Reply by davidfowl** [reply, marked as answer]
This tool might be helpful https://github.com/SaifAqqad/AspireRunner

**Reply by Nabakamal** [reply]
Ah! This is pretty cool - thanks. Let me give this one a go - cheers!
